releases/v19.2.0-rc.2.md

```diff
@@ -0,0 +1,102 @@
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.0-rc.2
+~~~
+
+### General changes
+
+- Go 1.12.10 is now required to build CockroachDB from source. [#41647][#41647] {% comment %}doc{% endcomment %}
+- Named ports in URLs are no longer supported due to a [URL parsing security fix in Go 1.12.8](https://github.com/golang/go/commit/3226f2d492963d361af9dfc6714ef141ba606713). Use numeric port numbers instead. [#41647][#41647] {% comment %}doc{% endcomment %}
+
+### SQL language changes
+
+- Mutations under [`UNION` or `UNION ALL`](../v19.2/selection-queries.html#union-combine-two-queries) are once again supported. [#41500][#41500] {% comment %}doc{% endcomment %}
+- Added new `skip` and `nullif` options for [`IMPORT ... DELIMITED DATA`](../v19.2/import.html), which ignore the specified number of lines at the beginning of the file and force all empty string fields to be inserted as `NULL` in the imported table, respectively. [#41656][#41656] {% comment %}doc{% endcomment %}
+- It is no longer possible to use [`IMPORT INTO`](../v19.2/import-into.html) with an interleaved table. This restriction is temporary and will be lifted in a future release. [#41651][#41651] {% comment %}doc{% endcomment %}
+- [`IMPORT`](../v19.2/import.html) is no longer allowed during a [rolling upgrade](../v19.2/upgrade-cockroach-version.html). [#41733][#41733] {% comment %}doc{% endcomment %}
+
+### Bug fixes
+
+- Fixed multiple bugs relating to `COPY FROM PARENT` in [replicaton zones](../v19.2/configure-replication-zones.html). [#41677][#41677]
+- The `system.replication_stats` table no longer incorrectly shows some ranges belonging to [table partitions](../v19.2/partitioning.html) as over-replicated. [#41700][#41700]
+- Fixed a bug that resulted in some queries returning an "index join must be against primary index" error. [#41723][#41723]
+- Fixed a bug where backfills could lead to certain [changefeeds](../v19.2/change-data-capture.html) row updates never being emitted. [#41721][#41721]
+- Fixed a bug that could cause [changefeeds](../v19.2/change-data-capture.html) row updates to be emitted at a timestamp less than or equal to an already forwarded resolved timestamp. [#41714][#41714]
+- Fixed a bug in cloud storage sink file naming for [changefeeds](../v19.2/change-data-capture.html) that could violate ordering in the presence of schema changes. [#41693][#41693]
+
+### Performance improvements
+
+- Increased [`BACKUP`](../v19.2/backup.html) parallelism in some cases. [#41729][#41729]
+- Improved the performance of generating the `system.replication_*` tables for large clusters. [#41757][#41757]
+- Improved performance of bulk ingestion. [#41768][#41768]
+
+### Doc updates
+
+- Clarified the [upgrade path for rolling upgrades](../v19.2/upgrade-cockroach-version.html), namely, that a cluster must be on a production release of the previous version, as opposed to a testing release (alpha/beta), to upgrade to the next major version. [#5644][#5644]
```

Comment:
Total nit but this would read more easily to me if the last clause came first, e.g., "to upgrade to the next major version, a cluster must be ..."